### PR TITLE
Fix: SDKs a/b test SSR hydration

### PR DESCRIPTION
**Pull request by samijaber**
## Description

SDKs:
- Vue: stop asyncly importing components (except those absolutely necessary to avoid circular dependencies). This was causing the A/B test inline styles not to render/update properly.
- A/B testing: fix hydration target check

### CODE REVIEW

File: packages/sdks/src/components/render-content-variants/render-content-variants.lite.tsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ export default function RenderContentVariants(props: VariantsProviderProps) {
         customComponents={props.customComponents}
         classNameProp={`variant-${props.content?.id}`}
         parentContentId={props.content?.id}
-        isSsrAbTest={state.shouldRenderVariants}
+        isSsrAbTest={false}
```
Comment (mrkoreye) [inline packages/sdks/src/components/render-content-variants/render-content-variants.lite.tsx:92]: Is this change intentional or leftover from testing something?

**Review by mrkoreye — APPROVED** [review]

File: packages/sdks/src/components/render-content-variants/render-content-variants.lite.tsx (hunk 1 above)
Comment (samijaber) [reply packages/sdks/src/components/render-content-variants/render-content-variants.lite.tsx:92]: It's a mistake which is causing the build to fail! Thanks for pointing this out to me 🙏🏽
